packages/react-composites/src/composites/MeetingComposite/adapter/MeetingAdapter.ts

```diff
@@ -14,15 +14,15 @@ import {
   ParticipantLeftListener
 } from '../../CallComposite';
 import { ChatAdapter, MessageReadListener, MessageReceivedListener, MessageSentListener } from '../../ChatComposite';
-import { MeetingState } from '../state/MeetingState';
+import { MeetingClientState } from '../state/MeetingState';
 
 export type ConflictingProps = 'getState' | 'onStateChange' | 'offStateChange' | 'on' | 'off';
 
 export interface MeetingAdapter extends Omit<ChatAdapter, ConflictingProps>, Omit<CallAdapter, ConflictingProps> {
```

Comment:
Parallel to my comment on #678: Explicitly list the constituents of the state with `Pick` rather than by negation with `Omit`.